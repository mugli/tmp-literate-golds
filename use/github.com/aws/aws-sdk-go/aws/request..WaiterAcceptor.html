<!DOCTYPE html>
<html>
<head>
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<title>References: github.com/aws/aws-sdk-go/aws/request.WaiterAcceptor</title>
<link href="../../../../../css/light-v0.3.2-preview.css" rel="stylesheet">
<script src="../../../../../jvs/golds-v0.3.2-preview.js"></script>
<body onload="onPageLoad()"><div>

<pre><code><span style="font-size:x-large;">type <b><a href="../../../../../pkg/github.com/aws/aws-sdk-go/aws/request.html">github.com/aws/aws-sdk-go/aws/request</a>.<a href="../../../../../src/github.com/aws/aws-sdk-go/aws/request/waiter.go.html#line-216">WaiterAcceptor</a></b></span>

<span class="title">3 uses</span>

	github.com/aws/aws-sdk-go/aws/request (current package)
		<a href="../../../../../src/github.com/aws/aws-sdk-go/aws/request/waiter.go.html#line-76">waiter.go#L76</a>: 	Acceptors []<b>WaiterAcceptor</b>
		<a href="../../../../../src/github.com/aws/aws-sdk-go/aws/request/waiter.go.html#line-216" class="path-duplicate">waiter.go</a><a href="../../../../../src/github.com/aws/aws-sdk-go/aws/request/waiter.go.html#line-216">#L216</a>: type <b>WaiterAcceptor</b> struct {
		<a href="../../../../../src/github.com/aws/aws-sdk-go/aws/request/waiter.go.html#line-226" class="path-duplicate">waiter.go</a><a href="../../../../../src/github.com/aws/aws-sdk-go/aws/request/waiter.go.html#line-226">#L226</a>: func (a *<b>WaiterAcceptor</b>) match(name string, l aws.Logger, req *Request, err error) (bool, error) {
</code></pre><pre id="footer">
<table><tr><td><img src="../../../../../png/go101-twitter.png"></td>
<td>The pages are generated with <a href="https://go101.org/article/tool-golds.html"><b>Golds</b></a> <i>v0.3.2-preview</i>. (GOOS=darwin GOARCH=amd64)
<b>Golds</b> is a <a href="https://go101.org">Go 101</a> project developed by <a href="https://tapirgames.com">Tapir Liu</a>.
PR and bug reports are welcome and can be submitted to <a href="https://github.com/go101/golds">the issue list</a>.
Please follow <a href="https://twitter.com/go100and1">@Go100and1</a> (reachable from the left QR code) to get the latest news of <b>Golds</b>.</td></tr></table></pre>