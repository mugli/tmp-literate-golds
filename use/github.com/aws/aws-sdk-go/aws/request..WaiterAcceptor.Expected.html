<!DOCTYPE html>
<html>
<head>
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<title>References: github.com/aws/aws-sdk-go/aws/request.WaiterAcceptor.Expected</title>
<link href="../../../../../css/light-v0.3.2-preview.css" rel="stylesheet">
<script src="../../../../../jvs/golds-v0.3.2-preview.js"></script>
<body onload="onPageLoad()"><div>

<pre><code><span style="font-size:x-large;"><b><a href="../../../../../pkg/github.com/aws/aws-sdk-go/aws/request.html">github.com/aws/aws-sdk-go/aws/request</a>.<a href="../../../../../src/github.com/aws/aws-sdk-go/aws/request/waiter.go.html#line-216">WaiterAcceptor</a>.<a href="../../../../../src/github.com/aws/aws-sdk-go/aws/request/waiter.go.html#line-220">Expected</a></b></span><span style="font-size: large;"><i> (field)</i></span>

<span class="title">5 uses</span>

	github.com/aws/aws-sdk-go/aws/request (current package)
		<a href="../../../../../src/github.com/aws/aws-sdk-go/aws/request/waiter.go.html#line-220">waiter.go#L220</a>: 	<b>Expected</b> interface{}
		<a href="../../../../../src/github.com/aws/aws-sdk-go/aws/request/waiter.go.html#line-239" class="path-duplicate">waiter.go</a><a href="../../../../../src/github.com/aws/aws-sdk-go/aws/request/waiter.go.html#line-239">#L239</a>: 			if !awsutil.DeepEqual(val, a.<b>Expected</b>) {
		<a href="../../../../../src/github.com/aws/aws-sdk-go/aws/request/waiter.go.html#line-248" class="path-duplicate">waiter.go</a><a href="../../../../../src/github.com/aws/aws-sdk-go/aws/request/waiter.go.html#line-248">#L248</a>: 			if awsutil.DeepEqual(val, a.<b>Expected</b>) {
		<a href="../../../../../src/github.com/aws/aws-sdk-go/aws/request/waiter.go.html#line-256" class="path-duplicate">waiter.go</a><a href="../../../../../src/github.com/aws/aws-sdk-go/aws/request/waiter.go.html#line-256">#L256</a>: 		s := a.<b>Expected</b>.(int)
		<a href="../../../../../src/github.com/aws/aws-sdk-go/aws/request/waiter.go.html#line-260" class="path-duplicate">waiter.go</a><a href="../../../../../src/github.com/aws/aws-sdk-go/aws/request/waiter.go.html#line-260">#L260</a>: 			result = aerr.Code() == a.<b>Expected</b>.(string)
</code></pre><pre id="footer">
<table><tr><td><img src="../../../../../png/go101-twitter.png"></td>
<td>The pages are generated with <a href="https://go101.org/article/tool-golds.html"><b>Golds</b></a> <i>v0.3.2-preview</i>. (GOOS=darwin GOARCH=amd64)
<b>Golds</b> is a <a href="https://go101.org">Go 101</a> project developed by <a href="https://tapirgames.com">Tapir Liu</a>.
PR and bug reports are welcome and can be submitted to <a href="https://github.com/go101/golds">the issue list</a>.
Please follow <a href="https://twitter.com/go100and1">@Go100and1</a> (reachable from the left QR code) to get the latest news of <b>Golds</b>.</td></tr></table></pre>